### ISSUE TITLE
cannot load such file -- ruby2_keywords (LoadError) on Ruby 2.6

### ISSUE THREAD

**Issue by mishina2228** [freerange/mocha#581]
I ran into a LoadError with mocha 2.0.0 and Ruby 2.6.

```
# ruby --version
ruby 2.6.10p210 (2022-04-12 revision 67958) [x86_64-linux-musl]
# irb
irb(main):001:0> require 'mocha/api'
Traceback (most recent call last):
       16: from (irb):1
       15: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:34:in `require'
       14: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:130:in `rescue in require'
       13: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:130:in `require'
       12: from /usr/local/bundle/gems/mocha-2.0.0/lib/mocha/api.rb:3:in `<top (required)>'
       11: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
       10: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        9: from /usr/local/bundle/gems/mocha-2.0.0/lib/mocha/hooks.rb:1:in `<top (required)>'
        8: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        7: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        6: from /usr/local/bundle/gems/mocha-2.0.0/lib/mocha/mockery.rb:2:in `<top (required)>'
        5: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        4: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        3: from /usr/local/bundle/gems/mocha-2.0.0/lib/mocha/mock.rb:1:in `<top (required)>'
        2: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        1: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
LoadError (cannot load such file -- ruby2_keywords)
```

ruby2_keywords seems to be bundled from Ruby 2.7.
Should it be added to runtime dependency on [mocha.gemspec](https://github.com/freerange/mocha/blob/c5cf3249d9706f3470cbfcfd76e97b4bae87a3d0/mocha.gemspec) rather than [Gemfile](https://github.com/freerange/mocha/blob/c5cf3249d9706f3470cbfcfd76e97b4bae87a3d0/Gemfile#L32)?

**Comment by floehopper** [comment]
@mishina2228 Thanks for reporting. Yes, I think you're right and I'm guessing our tests didn't catch it, because they include the bundled runtime dependencies. Sorry about that. I'll try to get that fixed as soon as I can.

**Comment by floehopper** [comment]
@wasabigeek Do you know if there was a specific reason for the `'~> 0.0.5'` on the `ruby2_keywords` gem?

**Comment by wasabigeek** [comment]
@floehopper only to tie to the current version of the gem since we've tested it to work.

Sorry about the gemspec, I missed that as well 🙇‍♂️

**Comment by floehopper** [comment]
@wasabigeek Thanks - no worries! 😄

**Comment by floehopper** [comment]
PR opened: #582

**Comment by floehopper** [comment]
@mishina2228 This change has been released in v2.0.1. Please let me know if you run into any problems with it. Thanks again for reporting it.
